package.json

```diff
@@ -34,7 +34,7 @@
     "eslint": "^4.6.1",
     "eslint-config-scratch": "^5.0.0",
     "gh-pages": "^1.0.0",
-    "jsdoc": "^3.5.5",
+    "jsdoc": "^3.6.0",
```

Comment:
Might want to add "resolves #680" to the PR description... at least I think that's what this is...?

Reply:
I broke this out into https://github.com/LLK/scratch-render/pull/683